modules/space/rendering/renderabletravelspeed.cpp

```diff
@@ -71,19 +72,32 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo IndicatorLengthInfo = {
         "IndicatorLength",
         "Indicator Length",
-        "This value specifies the length of the light indicator set in light seconds.",
+        "The length of the speed indicator line, given in seconds. The length will be "
+        "computed as the speed times this value. For example, a value of 1 will make it "
+        "as long as the distance it would travel over one second.",
         openspace::properties::Property::Visibility::User
     };
 
     constexpr openspace::properties::Property::PropertyInfo FadeLengthInfo = {
         "FadeLength",
         "Fade Length",
-        "This value specifies the length of the faded tail of the light indicator "
-        "set in light seconds.",
+        "The length of the faded tail of the speed indicator, given in seconds. The "
+        "length of the tail will be computed as the speed times this value. For example, "
+        "a value of 1 will make it as long as the distance it would travel over one "
+        "second. A linear fade will be applied over this distance to create the tail.",
         openspace::properties::Property::Visibility::User
     };
 
-    struct [[codegen::Dictionary(RenderableLightTravel)]] Parameters {
+    // This renderable can be used to visualize a certain travel speed, using a line that
+    // moves in the provided speed from a start object to a target. The start position
```

Comment:
```suggestion
    // moves in the provided speed from a start object to a target object. The start position
```

Reply:
Are you thinking of `Parent` ?  Would be great to discuss what we prefer in this case.  

Should it be `Parent`/`Target` (code), parent/target (lower-case), or Parent/Target (capitalized)?